scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -660,7 +661,11 @@ class Framework extends SbtFramework {
         }
       }
 
-    val poolSize = Runtime.getRuntime.availableProcessors * 2
+    val poolSize =
```

Comment:
can we check for -1 here and use ```Executors.newCachedThreadPool(threadPool)``` in this case?

Reply:
@drieks sorry for a year of delay in responding, but yes I agree that's a good suggestion, I have added it in the latest commit in this PR.